### ISSUE TITLE
Articles 

### ISSUE THREAD

**Issue by anti-conformiste** [opensupports/opensupports#655]
When using heading et normal format (h1-h6 and p tags), editor insert <p><br></p> each time saving article and content shifts down to bottom.

How to reproduce ?

1. Just create an article, insert header (it doesn't matter which) 
2. Insert a paragrah.
3. Insert another header.

**Comment by mredigonda** [comment]
Thank you for informing us!

It seems that the problem is a newline above the headers each time you save an article, is this correct?

This could probably be caused by the library we are using for the text editor.

**Comment by anti-conformiste** [comment]
indeed, I also think the updaute of the editor framework cause this issue. This bug did not exist before version 4.4.0
Here is [a link](https://transfer.sh/FBQRJ/OpenSupportsBugEditor.mp4) to see the problem in video.

**Comment by anti-conformiste** [comment]
I found a way to temporarily solve the issue waiting the fix. You can  comment line 166 to 173 in api/vendor/ezyang/htmlpurifier/library/HTMLPurifier/Generator.php file.
By doing this keep in mind all empty paragraph will be removed when saving article.
